bin/logstash.bat

```diff
@@ -33,24 +33,18 @@ for /F "usebackq tokens=1-2* delims= " %%A in (!params!) do (
 	)
 )
 
-rem if explicit jvm.options is not found use default location
-if "%LS_JVM_OPTIONS_CONFIG%" == "" (
-  set LS_JVM_OPTIONS_CONFIG="%LS_HOME%\config\jvm.options"
+for %%i in ("%LS_HOME%\logstash-core\lib\jars\*.jar") do (
+	call :concat "%%i"
 )
 
-rem extract the options from the JVM options file %LS_JVM_OPTIONS_CONFIG%
-rem such options are the lines beginning with '-', thus "findstr /b"
-if exist %LS_JVM_OPTIONS_CONFIG% (
-  for /F "usebackq delims=" %%a in (`findstr /b \- %LS_JVM_OPTIONS_CONFIG%`) do set options=!options! %%a
-  set "LS_JAVA_OPTS=!options! %LS_JAVA_OPTS%"
-) else (
-  echo "warning: no jvm.options file found"
```

Comment:
I think the changeset to this file would be much simpler and less risky if our `JvmOptionsParser` _required_ the path to a readable options file and we continued to do the finding of that file in the relevant platform-specific places.

Reply:
I thought that moving all the shell specific code into Java code could be safer and easier to manage, where probably we could also eventually cover with some unit testing.